components/prism-liquid.js

```diff
@@ -32,7 +32,11 @@ Prism.languages.liquid = {
 	// https://github.com/Shopify/liquid/blob/698f5e0d967423e013f6169d9111bd969bd78337/lib/liquid/lexer.rb#L21
 	'number': /\b\d+(?:\.\d+)?\b/,
 	'operator': /[!=]=|<>|[<>]=?|[|?:=-]|\b(?:and|or|contains(?=\s))\b/,
-	'punctuation': /[.,\[\]()]/
+	'punctuation': /[.,\[\]()]/,
+	'empty': {
+		pattern: /empty/,
```

Comment:
Always use boundary assertions. Otherwise, variables like `is_empty` will look funny.

```suggestion
		pattern: /\bempty\b/,
```

Reply:
That's what I was missing, thank you!